godtools/App/Features/AppLanguage/Domain/UseCases/ViewDownloadableLanguagesUseCase.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ViewDownloadableLanguagesUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 12/4/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class ViewDownloadableLanguagesUseCase {
+    
+    private let getInterfaceStringsRepository: GetDownloadableLanguagesInterfaceStringsRepositoryInterface
+    
+    init(getInterfaceStringsRepository: GetDownloadableLanguagesInterfaceStringsRepositoryInterface) {
+        
+        self.getInterfaceStringsRepository = getInterfaceStringsRepository
+    }
+    
+    func viewPublisher(appLanguage: AppLanguageDomainModel) -> AnyPublisher<ViewDownloadableLanguagesDomainModel, Never> {
+        
+        return self.getInterfaceStringsRepository.getStringsPublisher(translateInAppLanguage: appLanguage).map {
```

Comment:
Hmm ```map``` is a bit cleaner here, I probably should be using ```map``` as well instead of ```flatMap```.  I'll see about changing some of the UseCases to use ```map``` instead.